### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#446] [Esami & Esoneri M-Z]
### Traccia

<img width="667" alt="image" src="https://github.com/user-attachments/assets/eddf92c3-8b2d-42ac-ac43-7f6b8ae077ee">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/01ec0956-821d-4866-9134-2e6196d35780)
